include/sdf/JointAxis.hh

```diff
@@ -229,6 +229,16 @@ namespace sdf
     /// \return SDF element pointer with updated joint values.
     public: sdf::ElementPtr ToElement(unsigned int _index = 0u) const;
 
+    /// \brief Create and return an SDF element filled with data from this
+    /// joint axis.
+    /// Note that parameter passing functionality is not captured with this
+    /// function.
+    /// \param[out] _errors Vector of errors.
+    /// \param[in] _index Index of this joint axis
+    /// \return SDF element pointer with updated joint values.
+    public: sdf::ElementPtr ToElement(sdf::Errors &_errors,
```

Comment:
I see `JointAxis_TEST` does not have a `ToElement` test so far, could you add some together with this overload?

Reply:
Added a test for `ToElement`.